geode-core/src/distributedTest/java/org/apache/geode/internal/cache/ClientServerTransactionDUnitTest.java

```diff
@@ -2076,10 +2078,11 @@ public Object call() throws Exception {
       }
     });
 
-    client.invoke(new SerializableCallable() {
+    TXId txid2 = (TXId) client.invoke(new SerializableCallable() {
```

Comment:
There is no need to have a new txId here, unless you want to assert this is the same as txId. There is only one transaction. Probably just ignore the return value here. And later resume the original txId instead of txid2.